### PR TITLE
Optional .deploy extension

### PR DESCRIPTION
**Pull request by ben-farrant**
Some clickonce publish folders do not use .deploy file extension. Adding .deploy breaks the app.

### CODE REVIEW

File: updateClickOnce/task.json [hunk 1]
```diff
@@ -91,6 +91,14 @@
         "required": false,
         "helpMarkDown": "The minimum version of this application a user can run. This flag makes the named version of your application a required update, use %version% to use the current version as minimum. The argument must be a valid version string of the format N.N.N.N"
       },
+      { 
+        "name": "ReplaceDeploy",
+        "type": "bool",
+        "label": "Replace Deploy File Extension",
+        "defaultValue": false,
```
Comment (phelje) [inline updateClickOnce/task.json]: Maybe it’s best to have this as default true to maintain comp ability to current version.

**Comment by phelje** [comment]
You know what!! this is a great idea but ill make this automatic.

File: updateClickOnce/task.json (hunk 1 above)
Comment (ben-farrant) [reply updateClickOnce/task.json]: Updated to default to true

**Comment by ben-farrant** [comment]
Doing it automatically for any files with .deploy extension.

**Comment by phelje** [comment]
Have you hade time to test the new version, is it working ok for you?

**Comment by ben-farrant** [comment]
> Have you hade time to test the new version, is it working ok for you?

Sorry it's taken time to respond. Yes it worked great.

**Pull request closed without merging**